Content.Shared/_Moffstation/Traits/Assorted/UncloneableComponent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Robust.Shared.GameStates;
+
+namespace Content.Shared._Moffstation.Traits.Assorted;
+
+/// <summary>
+/// This is used for the uncloneable trait.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class UncloneableComponent : Component
+{
+    /// <summary>
+    /// Can this player be cloned using a cloning pod?
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public bool Cloneable = false;
+}
```

Comment:
```suggestion
public sealed partial class UncloneableComponent : Component;
```
Nothing ever modifies this from `false`, so IMO this should just be a marker component whose presence implies uncloneability.